bigframes/core/global_session.py

```diff
@@ -36,7 +40,15 @@ def close_session() -> None:
 
     with _global_session_lock:
         if _global_session is not None:
-            _global_session.close()
+            try:
+                _global_session.close()
+            except google.auth.exceptions.RefreshError as e:
+                session_id = _global_session.session_id
+                warnings.warn(
+                    "Session cleanup failed for session with"
+                    + "id: {session_id}".format(session_id=session_id)
+                )
+                print(e)
```

Comment:
Nit: This should be in stderr. Also, just printing the exception can lose valuable debugging info. Let's do the whole traceback.

https://stackoverflow.com/a/16946886/101923

```suggestion
                traceback.print_tb(err.__traceback__)
```

In future, we can do `traceback.print_exception` (https://docs.python.org/3/library/traceback.html#traceback.print_exception) but if I'm reading that right not until Python 3.10 is our minimum version.